src/libraries/System.Private.CoreLib/src/System/Resources/ResourceSet.cs

```diff
@@ -18,7 +18,7 @@ namespace System.Resources
     //
     public class ResourceSet : IDisposable, IEnumerable
     {
-        protected IResourceReader Reader = null!;
+        private IResourceReader? _reader;
```

Comment:
This is another one of those potential breaking changes for .NET Framework binaries. This field is part of .NET Framework public surface: https://learn.microsoft.com/en-us/dotnet/api/system.resources.resourceset.reader?view=netframework-4.8

Reply:
Got it & fixed